doc/helm/dev/developer_guide.md

```diff
@@ -118,36 +162,44 @@ spec:
           resources:
             {}
 
-
----
-# Source: foo/templates/ingress.yaml
-
+NOTES:
```

Comment:
This block should be outside of the YAML example correct?

Reply:
No, that's part of the output of `helm install --dry-run`